tests/kernels/test_flash_attn.py

```diff
@@ -105,10 +106,10 @@ def test_flash_attn_with_paged_kv(
                                  (num_seqs, max_num_blocks_per_seq),
                                  dtype=torch.int32)
 
-    output = flash_attn_with_kvcache(
-        q=query.unsqueeze(1),
-        k_cache=key_cache,
-        v_cache=value_cache,
+    output = torch.ops.vllm.flash_attn_with_kvcache(
```

Comment:
I think it would be good to add some `torch.library.opcheck` tests here on top of testing the ops.

Reply:
added in [fc2a4c2](https://github.com/vllm-project/vllm/pull/7536/commits/fc2a4c2898159885c21441217deb2373a9776512)